psi/pmt_test.go

```diff
@@ -47,6 +47,10 @@ func (es *testPmtElementaryStream) StreamType() uint8 {
 	return es.streamType
 }
 
+func (es *testPmtElementaryStream) StreamTypeDescription() string {
+	return ""
```

Comment:
Should we return some test content here like "Test Stream Description" here?

Reply:
Sure that's a good idea.

Reply:
Done.